templates/indicators/indicator_form_modal.html

```diff
@@ -287,7 +287,8 @@ <h3 id="id_subtitle" class="no-bold indicator_name">
 
                         <div class="form-group" id="parent_div_id_direction_of_change">
                             <div class="" id="fieldgroup_id_direction_of_change">
-                                {% render_field form.direction_of_change class+="form-control mt-2" style="width:100%; border-radius: 0%" %}
+                                <label for="id_direction_of_change">{{ form.direction_of_change.label }}</label>
+                                {% render_field form.direction_of_change class+="form-control" style="width:100%; border-radius: 0%" %}
```

Comment:
the style attribute should not be necessary here

Reply:
Ack, yeah, a little too much copy pasta.  Thanks for catching.